sdk/schemaregistry/azure-schemaregistry-avroserializer/azure/schemaregistry/serializer/avroserializer/_schema_registry_avro_serializer.py

```diff
@@ -76,8 +75,9 @@ def close(self):
         """
         self._schema_registry_client.close()
 
-    def _get_schema_id(self, schema_name, schema, **kwargs):
-        # type: (str, avro.schema.Schema, Any) -> str
+    @lru_cache(maxsize=128)
```

Comment:
shouldn't this 128 be a variable that is configurable by the user?

Reply:
so I believe for GA, this doesn't need to be configurable. I can move it out so a separate variable when we do decide to make it configurable. Thoughts?